src/commands/general/ping.ts

```diff
@@ -1,6 +1,6 @@
 import type { Command } from "../../types/Command";
```

Comment:
you have custom import paths like `#types` in your package.json, use it

Reply:
future PR